2_observations.yml

```diff
@@ -72,6 +87,10 @@ targets:
     
   2_observations/out/obs_temp_drb.rds:
     command: gd_get('2_observations/out/obs_temp_drb.rds.ind')
+    
+  # ngwos reaches that should be withheld after 2017-10-01 for NGWOS experiment
+  ngwos_reaches:
```

Comment:
Am I right that you get these, but you don't write them out to a file?

Reply:
Yeah this is just an object, don't write as a file. But I can if that would be useful?